src/Router.tsx

```diff
@@ -207,6 +207,7 @@ const router = createBrowserRouter(
             projectPath + sep + PROJECT_ENTRYPOINT
           )
           const children = await readDir(projectPath, { recursive: true })
+          kclManager.setCodeAndExecute(code, false)
```

Comment:
this replaces https://github.com/KittyCAD/modeling-app/pull/897/files#diff-26ad4b834941d9b19ebf9db8082bd202aaf72ea0ddea85f5a8a0cb3c729cc6f2L86-L104